docs/source/routing/performance/caching/in-memory.mdx

```diff
@@ -80,6 +80,17 @@ then look at `apollo_router_schema_loading_time` and `apollo.router.query_planni
 
 If the router is using distributed caching for query plans, the warm-up phase will also store the new query plans in Redis. Since all Router instances might have the same distributions of queries in their in-memory cache, the list of queries is shuffled before warm-up, so each Router instance can plan queries in a different order and share their results through the cache.
 
+#### Cache warm-up with headers
+
+<MinVersionBadge version="Router v1.61.0" />
+
+With router v1.61.0+  and v2.x+, if you have enabled exposing query plans via `--dev` mode or `plugins.experimental.expose_query_plan: true`, you can pass the `Apollo-Expose-Query-Plan` header to return query plans in the GraphQL response extensions. You must set the header to one of the following values:
+
+- `true`: Returns a human-readable string and JSON blob of the query plan while still executing the query to fetch data.
+- `dry-run`: Generates the query plan and aborts without executing the query.
+
+After using `dry-run`, query plans are saved to your configured cache locations. Using real, mirrored, or similar to production operations is a great way to warmup the caches before transitioning traffic to new router instances.
+
```

Comment:
So this section was added at some point in v2 docset but never backported to v1?

Reply:
Indeed, looks like it was missed: https://www.apollographql.com/docs/graphos/routing/v1/performance/caching/in-memory#cache-warm-up